app/i18n.ts

```diff
@@ -1,28 +1,26 @@
-import i18n from 'i18next';
+import i18next from 'i18next';
 import { initReactI18next } from 'react-i18next';
 
-import enUS from '../locales/enUS.json';
+import enUS from '../locales/en-US.json';
 
-const resources = {
-  enUS,
-};
+const newInstance = i18next.createInstance();
 
-i18n
-  // pass the i18n instance to react-i18next.
+newInstance // pass the i18n instance to react-i18next.
   .use(initReactI18next)
-  // init i18next
   .init({
     debug: true,
-    fallbackLng: 'enUS',
+    fallbackLng: 'en-US',
```

Comment:
For i18n, languages should always be specified as `nn-NN` -- language code first, in lowercase, then a hyphen, and finally country code second, in uppercase.

Reply:
SGTM!